changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 10th October 2022
+
+* Added new operations [Get all source assignments](../operations/sources.md#get-all-source-assignments) and [Get all sources](../operations/sources.md#get-all-sources) for retrieval of reservation sources.
```

Comment:
- The link needs updating here, now that `Get all source assignments` has been moved to new file sourceassignments.md
- You might also want to update the Changelog date